core/tests/test_utils.py

```diff
@@ -2260,7 +2250,7 @@ def _parse_json_response(self, json_response, expect_errors):
 
         return json.loads(json_response.body[len(feconf.XSSI_PREFIX):])
 
-    def get_json(self, url, params=None, expected_status_int=200):
+    def get_json(self, url, params=None, expected_status_int=200, headers=None):
```

Comment:
Please add docstring.

Why do we need headers now, when we didn't before?

Reply:
Removed, it wasn't needed actually.